src/nostr-post/nostr-post.ts

```
@@ -0,0 +1,825 @@
+  event.stopPropagation();
+  });
+  });
+  }
+
+  async render() {
+  const content = this.post?.content ||  '';
+  const parsedContent = await this.parseText(content);
+  const htmlToRender = this.renderContent(parsedContent);
+
+  let date = '';
+  if(this.post && this.post.created_at) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)